### DISCUSSION TITLE
Poor documentation on https://reactstrap.github.io

### DISCUSSION THREAD

**Discussion by stsiushkevich** [reactstrap/reactstrap#2509] [Q&A]
Hi, Guys! The previous version of the documentation was comprehensive: https://deploy-preview-2356--reactstrap.netlify.app/. But the current documentation on https://reactstrap.github.io almost doesn't have useful information. Maybe are there more informable docs anywhere?

**Comment by TheSharpieOne** [comment]
Previous feedback was that the docs sucked, they were confusing and "non-standard" (Storybook being the defacto standard) so the docs were migrated to storybook. The information within the docs are more/less the same (just the order is different (the prop definitions are at the bottom)). It has examples with code and lists of props and what they do (this part is definitely more clear than before.)

Is there something specific that is missing?

**Reply by stsiushkevich** [reply]
Ok, I understood

**Reply by illiteratewriter** [reply]
I noticed a couple of minor issues with Storybook. Storybook does not show content other than JSX, which is an issue I think. Also, when you pass a function as prop to JSX, the content of the function is either hidden or converted into `() => {}` which is creating confusion for users.

**Comment by illiteratewriter** [comment]
@davidacevedo @TheSharpieOne Adding more text content to the docs to make it look more like the Bootstrap documentation might make it easier to use the documentation. Currently, we have examples in stories, with almost zero text and I think adding text might make things better.

For example [the accordion flush page for Reactstrap](https://reactstrap.github.io/?path=/docs/components-accordion--accordion#flush) doesn't say anything. I was thinking of taking the text from [Bootstrap accordion flush](https://getbootstrap.com/docs/5.1/components/accordion/#flush) and adding to it to look something like the following:

> Add `flush` prop to remove the default background-color, some borders, and some rounded corners to render accordions edge-to-edge with their parent container.

This would make it easier for users to use Reactstrap without referring to the bootstrap docs. Let me know your thoughts.

**Reply by illiteratewriter** [reply]
Also, props do not have a description of what they do, which is causing confusion as illustrated by this discussion(#2372). Just adding comments to props while writing the components will add the comment automatically to Storybook. So maybe we can add this too.

**Reply by illiteratewriter** [reply]
I've gone ahead and made a [PR](https://github.com/reactstrap/reactstrap/pull/2544) for Spinner to demonstrate the above.

**Reply by davidacevedo** [reply]
I think this looks great and we should do that for future props. I also enjoy the added test in storybook!
